### DISCUSSION TITLE
Where did the Bluestacks cask go?

### DISCUSSION THREAD

**Discussion by cyberdynesoftware** [Homebrew/discussions#766] [Casks]
I've just installed the Bluestacks cask, but there is no Bluestacks app in Applications nor is there a Bluestacks binary available on the command line. Something was downloaded though. Quite a bit actually. Where did it go? How to access it?

**Comment by SMillerDev** [comment]
Did it open an installer?

**Reply by cyberdynesoftware** [reply]
I did not notice anything opening.

Should it? Should I reinstall it? Or is there another way to trigger the installer?
